source/partials/release_notes/_release-16-1-0.html

```diff
@@ -0,0 +1,38 @@
+<h4>What's new in GoCD 16.1</h4>
+<p>
+  We are moving to a more regular release schedule, and so you might see releases with mostly issues fixed, while bigger level features are in progress across releases.
+</p>
+<h5>Important features and bug fixes in this release -</h5>
+<ul>
+  <li>
+    <a href="https://github.com/gocd/gocd/pull/1739">Don't send 'Server:' header from Jetty</a>
+  </li>
+  <li>
+    <a href="https://github.com/gocd/gocd/pull/1489">GoCD 15.2's config schema (cruise-config.xsd)
+      causes validation errors</a>
+  </li>
+  <li>
+    <a href="https://github.com/gocd/gocd/pull/1740">Throwing BadCredentialsException when user is not
+      found in configured…</a>
+  </li>
+  <li>
+    <a href="https://github.com/gocd/gocd/pull/1741">Pipeline Config validation fixes</a>
+  </li>
+  <li>
+    <a href="https://github.com/gocd/gocd/issues/1753">Pipeline API: updating a lot of pipelines stops
+      working </a>
+  </li>
+  <li>
+    <a href="https://github.com/gocd/gocd/pull/1779">Check approvers properly against allowed roles</a>
+  </li>
+  <li>
+    <a href="https://github.com/gocd/gocd/pull/1786">Fix some compatibility when installing RPM packages
+      on RHEL5 and derivatives (#1766) </a>
+  </li>
+</ul>
+<h4>Note</h4>
+<p>
+  The comprehensive list of changes for this release can be found <a href="https://github.com/gocd/gocd/pulls?q=is%3Apr+milestone%3A%22Release+16.1%22+is%3Aclosed">here.</a>
+</p>
+<h4>Thanks!</h4>
```

Comment:
I guess these sort of things call for specific helper methods —

`github_pr(42, "Add support for flux capacitor")` should become `<a href="https://github.com/gocd/gocd/pull/42">Add support for flux capacitor</a>`

`github_issue(42, "Flux capacitor is not fluxing")` should become `<a href="https://github.com/gocd/gocd/issue/42">Flux capacitor is not fluxing</a>`

`github_user('jdoe', 'John Doe')` should become `<a href='https://github.com/jdoe'>John Doe</a>`

`change_list('Release 16.1')` should become `<a href="https://github.com/gocd/gocd/pulls?q=is%3Apr+milestone%3A%22Release+16.1%22+is%3Aclosed"></a>`